src/cases/models/organisation/organisation.module.ts

```diff
@@ -0,0 +1,7 @@
+import { UserDetails } from '../user-details/user-details.module';
+
+export interface OrganisationModule {
```

Comment:
Module or Model? I think the filename as well as the interface name for UserDetail is also wrong.